templates/default/upstart.conf.erb

```diff
@@ -0,0 +1,52 @@
+# <%= @name %> generated by poise-service for <%= @new_resource.to_s %>
```

Comment:
might be nice to add a similar comment to the top of our sysvinit config as well